Azure/azure-cli
# {RDBMS} CLI code updates for new GA Api version for PostgreSQL using new SDK package

**Related command**
No new commands added/updated in this PR. Only updated package to use the latest Python SDK package.

Added sample commands for creating/updating postgresql flexible server with aad auth Enabled. (AD auth is disabled by default and password auth is enabled by default):

- `az postgres flexible-server create/update --active-directory-auth [Enabled/Disabled] --password-auth [Enabled/Disabled]`

**Description**

This PR uses latest Python SDK package which in turn uses the latest GA API version 2022-12-01. This PR will help our customers to start using new features like Data encryption, Active Directory Authentication, Geo Restore, Read Replica, Databases, Same zone HA, Fastest Restore point using Azure CLI.

Added new test to test active directory only auth.

**Testing Guide**
See examples in _helptext_pg.py and updated tests.




---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).
